docs/docs/sources/introduction.md

```diff
@@ -24,7 +24,11 @@ One of the crucial features Airy Core provides is the ability to process
 conversational data from a variety of sources (like Facebook Messenger, Google
 Business Messages, and so on).
 
-Connect sources programmatically with our sources guides or simply by clicking with our Channels UI.
+You can connect sources through API requests or using our [Channels UI](/ui/channels). Our sources guides covers both options, step-by-step.
+
+It's important to understand the difference between a [source](/getting-started/glossary/#source) and a [channel](/getting-started/glossary/#channel). A [channel](/getting-started/glossary/#channel) represents a connection between a [source](/getting-started/glossary/#source) and the Airy Core Platform: multiple [channels](/getting-started/glossary/#channel) can thus use the same [source](/getting-started/glossary/#source) for different [conversations](/getting-started/glossary/#conversation). When a [channel](/getting-started/glossary/#channel) is connected, a [conversation](/getting-started/glossary/#conversation) is created between the [source](/getting-started/glossary/#source) and the Airy Core instance.
```

Comment:
```suggestion
It's important to understand the difference between a [source](/getting-started/glossary/#source) and a [channel](/getting-started/glossary/#channel). A [channel](/getting-started/glossary/#channel) represents a connection between a [source](/getting-started/glossary/#source) and you Airy Core instance: multiple [channels](/getting-started/glossary/#channel) can thus use the same [source](/getting-started/glossary/#source) for different [conversations](/getting-started/glossary/#conversation). When a [channel](/getting-started/glossary/#channel) is connected, a [conversation](/getting-started/glossary/#conversation) is created between the [source](/getting-started/glossary/#source) and your Airy Core instance.
```

`When a [channel](/getting-started/glossary/#channel) is connected, a [conversation](/getting-started/glossary/#conversation) is created between the [source](/getting-started/glossary/#source) and the Airy Core instance`

While I corrected this, I don't think this is accurate. It reads like we create conversation on channel connection but we don't. What are we trying to say here? (honest question... I'm not sure I get it)

Reply:
yeah I had a hard time finding the right word to explain the process .I guess what's happening is that connecting a channel gives the possibility of starting a conversation. A conversation is created once the channel has been connected and as soon as a message has been sent between the Airy Core instance and the source.